layers/+irc/erc/packages.el

```diff
@@ -214,7 +214,8 @@
   (use-package erc-image
     :init (with-eval-after-load 'erc (add-to-list 'erc-modules 'image))))
 
-(defun erc/init-erc-terminal-notifier ())
+(defun erc/init-erc-terminal-notifier ()
+  (use-package erc-terminal-notifier))
```

Comment:
I think something like this should work (not tested though):

```bash
$ git checkout erc-terminal-notifier-fix
$ git checkout -b temp-branch-just-in-case-it-goes-wrong
$ git checkout erc-terminal-notifier-fix
$ git reset --hard develop
$ git cherry-pick 84388fb9ddca5466b556c7fcc94bb01d6f199643
$ git push -f

$ # If everything went fine:
$ git branch -D temp-branch-just-in-case-it-goes-wrong
```

I haven't tested it, so be sure to keep the diff of your changes somewhere (`git
diff`) in case everything goes screwed up. For the push you may have to specify the remote depending on your configuration.

BTW it happened because you created the new branch from your `pdf-tools` branch
instead of `default` :-)